services/scorecard/scorecard.service.js

```diff
@@ -0,0 +1,41 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  score: Joi.required(),
+}).required()
+
+export default class ScorecardScore extends BaseJsonService {
+  static category = 'analysis'
+
+  static route = { base: 'scorecard', pattern: ':host/:orgName/:repoName' }
```

Comment:
Does the upstream service support branch-specific scans or is it only at the repo level?

Is there a fixed set of providers it integrates with (e.g. GitHub, GitLab, Bitbucket, etc.) or is it truly open ended?

Reply:
The scans are branch specific. Currently, it will only be integrated with GitHub, but there are future plans to expand it to GitLab and BitBucket

Reply:
The scans are branch specific. Currently, it will only be integrated with GitHub, but there are future plans to expand it to GitLab and BitBucket